tools/nuitka/build_nuitka

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+# This is a trial error to overcome a problem when using --recurse-all and then
+# inspect.isfunction starting to puke on majority of our interfaces
+
+# --recurse-stdlib  leads to some "fancy" errors
+
+#nuitka --exe --python-flag=no_site --recurse-directory=datalad --recurse-to={datalad{,.distribution,.support,.interface},wrapt,six,git,gitdb,argcomplete,iso8601,requests,keyring,appdirs,msgpack,smmap,tqdm,pyld,colorama,docopt,patoolib,urllib3,chardet,_patool_configdata,pkg_resources,backports.ssl_match_hostname,scrapy,cssselect,cssselect.xpath,lxml,lxml.html,w3lib.url,w3lib.http,w3lib.html,w3lib{,.encoding,.util},simplejson,nibabel,humanize,yaml,httpretty,github,jsmin,h5py,bs4,mock,nose,numpy} --recurse-not-to={boto,gtk,pyglet,tornado,matplotlib,twisted,numpy,pandas,vcr,sphinx,zmq,PIL,IPython,OpenGL} `which datalad` 2>&1 | tee $0.log
+
+set -eu
+
+os=linux
```

Comment:
`os` needs to be a parameter to the script, no?

Reply:
The --exe is the default, you can remoe it.